### ISSUE TITLE
Calling riskpool.getBundleInfo on burnt bundle fails with ERC721: invalid token ID

### ISSUE THREAD

**Issue by doerfli** [etherisc/depeg-contracts#24]
```
FAILED tests/test_policy_lifecycle.py::test_happy_path - brownie.exceptions.VirtualMachineError: revert: ERC721: invalid token ID```

**Comment by doerfli** [comment]
Testcase `tests/test_policy_lifecycle.py::test_happy_path` on branch `bugfix/burnt_bundle` extended to make it fail

**Comment by matthiaszimmermann** [comment]
fixed, see pull request #25
